lib/gringotts/gateways/pin_pay.ex

```diff
@@ -0,0 +1,423 @@
+defmodule Gringotts.Gateways.Pinpay do
+  @moduledoc """
+  [PinPay][home] gateway implementation.
```

Comment:
Be consistent whenever you refer to your gateway in your docs, you should follow their branding. In this case, it should be `PinPayments`.